nuxeo-connect-client/src/test/java/org/nuxeo/connect/pm/tests/TestPackageDependencyResolutionP2CUDF.java

```diff
@@ -57,59 +57,67 @@ protected void tearDown() throws Exception {
         CUDFHelper.defaultAllowSNAPSHOT = false;
     }
 
+    // Before: [AA-1.0.0, BB-1.0.0, Blocker-1.0.0, Blocker2-1.0.0, EE-1.0.0, NXBT-654.1-1.0.0, NXBT-654.2-1.0.0,
```

Comment:
This class makes use of deprecated `DummyPackageSource` constructor. Since it's an easy fix, it is recommended (best practice) to take the chance and reduce the technical debt while editing that file.